source/user-manual/api/getting-started.rst

```diff
@@ -24,78 +24,224 @@ The API starts at boot time. To control or check the **wazuh-api** service, use
+
+    .. code-block:: none
+        :class: output
+
+        {"token": "<YOUR_JWT_TOKEN>"}
+
+2. Run any endpoint following this structure. Replace <endpoint> and <YOUR_JWT_TOKEN> with your values.
+
+    .. code-block:: console
+
+        # curl -X <METHOD> "https://localhost:55000/v4/<ENDPOINT>" -H  "accept: application/json" -H  "Authorization: Bearer <YOUR_JWT_TOKEN>"
+
+Send a *request* to confirm that everything is working as expected:
 
 .. code-block:: console
 
-    # curl -u foo:bar "http://localhost:55000?pretty"
+    curl -X GET "https://localhost:55000/v4/" -H  "accept: application/json" -H "Authorization: Bearer <YOUR_JWT_TOKEN>"
 
 .. code-block:: json
     :class: output
 
     {
-        "error": 0,
-        "data": {
-            "msg": "Welcome to Wazuh HIDS API",
-            "api_version": "v|WAZUH_LATEST|",
-            "hostname": "wazuh",
-            "timestamp": "Mon Dec 03 2018 00:36:13 GMT+0000 (UTC)"
-        }
+        "title": "Wazuh API",
+        "api_version": "4.0.0",
+        "revision": 4000,
+        "license_name": "GPL 2.0",
+        "license_url": "https://github.com/wazuh/wazuh/blob/master/LICENSE",
+        "hostname": "wazuh-master",
+        "timestamp": "2020-05-25T07:05:00+0000"
     }
 
 
-Explanation:
-
- * ``curl``: A command-line tool for sending requests and commands over HTTP and HTTPS.
- * ``-u foo:bar``: The username and password to authenticate with the API.
- * ``http://localhost:55000``: The API URL to use if you are running the command on the manager itself.
- * ``?pretty``: The parameter that makes the JSON output more human-readable.
-
 Basic concepts
 --------------
 
 Here are some of the basic concepts related to making API requests and understanding their responses:
 
-* The *base URL* for each request is ``https://IP:55000/`` or ``http://IP:55000/``, depending on whether or not SSL is enabled and set up in the API.
-* All responses are in *JSON format* with the following structure:
+-  The *cURL command* for each request contains:
+
+    +-------------------------------------------------+----------------------------------------------------------------------------------------+
+    | Field                                           | Description                                                                            |
+    +=================================================+========================================================================================+
+    | ``-X GET/POST/PUT/DELETE``                      | Specifies a custom request method to use when communicating with the HTTP server.      |
+    +-------------------------------------------------+----------------------------------------------------------------------------------------+
+    | ``https://localhost:55000/v4/<ENDPOINT>``       | The API URL to use if you are running the command on the manager itself.               |
```

Comment:
Expand this to specify the user must use HTTP or HTTPS depending on whether or not SSL is enabled and set up in the API.